db/dcrpg/types_online_test.go

```diff
@@ -208,14 +154,14 @@ func TestTimeRoundTripIncorrectValuerTimeDefLocal(t *testing.T) {
 	// Using dbtypes.TimeDefLocal, where Value ensures Local, stores the
 	// incorrect time.
 	var id uint64
-	err := db.QueryRow(`INSERT INTO testing (timestamp) VALUES ($1) RETURNING id;`,
+	err := sqlDb.QueryRow(`INSERT INTO testing (timestamp) VALUES ($1) RETURNING id;`,
```

Comment:
`INSERT INTO testing (timestamp) VALUES ($1) RETURNING id;` and `SELECT timestamp FROM testing;` sql statements seem to be repeated quite often. Maybe it would be cleaner if the statements were referenced through variables e.g.
```golang
const(
   insertStatement = "INSERT INTO testing (timestamp) VALUES ($1) RETURNING id;"
   fetchStatement = "SELECT timestamp FROM testing;"
)
```

Reply:
Good call.  Done.